src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimator.h

```diff
@@ -83,6 +83,21 @@ class FloatingBaseEstimator : public BipedalLocomotion::System::Advanceable<Floa
                            iDynTree::Transform& IMU_H_l_foot,
                            iDynTree::Transform& IMU_H_r_foot);
 
+        /**
+        * Get relative pose between IMU and the feet
+        * @param[in] encoders joint positions through encoder measurements
+        * @param[out] IMU_H_l_foot pose of the left foot contact frame with respect to the IMU frame
+        * @param[out] IMU_H_r_foot pose of the right foot contact frame with respect to the IMU frame
+        * @param[out] J_IMULF explicit Jacobian of left foot frame with respect to IMU frame
+        * @param[out] J_IMURF explicit Jacobian of right foot frame with respect to IMU frame
+        * @return True in case of success, false otherwise.
+        */
+        bool getIMU_H_feet(const iDynTree::JointPosDoubleArray& encoders,
+                           iDynTree::Transform& IMU_H_l_foot,
+                           iDynTree::Transform& IMU_H_r_foot,
+                           iDynTree::MatrixDynSize& J_IMULF,
+                           iDynTree::MatrixDynSize& J_IMURF);
```

Comment:
Aren't we supposed to use ``Eigen`` and ``manif`` objects in the main headers?

Reply:
Technically, yes. But since this ModelComputations class is specific to the usage of `iDynTree Model`, I stuck to the use of iDynTree objects within this class. 

It was a design decision. Do you think it should rather use Eigen or `manif` objects?

Reply:
If it is a public method, then probably it is better to stick to ``Eigen``/``manif`` for consistency. If instead this is an implementation specific to iDynTree, it may make sense to remain with iDynTree.

Reply:
Also, if this complicates a lot some of your experiments, we can also postpone this change.

Reply:
I will open an issue regarding this. So that these changes might be in place within a newer PR.